### PR TITLE
Added seamless boolean attribute for iframe

### PR DESCRIPTION
**Pull request by RitwikGupta**
Followed guidelines (I think) for the CHANGES and AUTHORS file

### CODE REVIEW

**Comment by willkg** [comment]
@RitwikGupta I apologize that this has been sitting around for so long. Can you take some time to update it? If not, let me know and I can do that. Thank you!

**Comment by RitwikGupta** [comment]
That should resolve the merge conflicts. My CHANGES commit may be incorrect, sorry about that! Unsure of the format.

**Comment by willkg** [comment]
This is failing for Python 3.4 test runs with weirdo errors. I'm going to try some Travis fiddling and if that doesn't work, look into it more carefully.

File: html5lib/constants.py [hunk 1]
```diff
@@ -588,24 +588,25 @@
...
+    "hr": frozenset(("noshade")),
+    "menu": frozenset(("autosubmit",)),
+    "fieldset": frozenset(("disabled", "readonly")),
+    "option": frozenset(("disabled", "readonly", "selected")),
+    "optgroup": frozenset(("disabled", "readonly")),
+    "button": frozenset(("disabled", "autofocus")),
+    "input": frozenset(("disabled", "readonly", "required", "autofocus", "checked", "ismap")),
+    "select": frozenset(("disabled", "readonly", "autofocus", "multiple")),
+    "output": frozenset(("disabled", "readonly")),
+    "iframe": frozenset(("seamless")),
```
Comment (willkg) [inline html5lib/constants.py]: Two things:

First, you changed all the lists to tuples. I'm not sure why you did that. One problem tuples has is that in order to make something a one-item tuple, you need a comma. Forgetting it causes problems ...

Which brings me to the second thing--this one is missing that comma. So this is creating a frozenset of `['s', 'e', 'a', 'm', 'l', 'e', 's', 's']` which is not what we want.

Can you fix this? I'd also prefer it if you switched these back to lists.

File: html5lib/constants.py [hunk 2]
```diff
@@ -588,24 +588,25 @@
...
+    "": frozenset(("irrelevant",)),
+    "style": frozenset(("scoped",)),
+    "img": frozenset(("ismap",)),
+    "audio": frozenset(("autoplay", "controls")),
+    "video": frozenset(("autoplay", "controls")),
+    "script": frozenset(("defer", "async")),
+    "details": frozenset(("open",)),
+    "datagrid": frozenset(("multiple", "disabled")),
+    "command": frozenset(("hidden", "disabled", "checked", "default")),
+    "hr": frozenset(("noshade")),
```
Comment (willkg) [inline html5lib/constants.py]: This one is also missing a comma.

**Comment by willkg** [comment]
Travis fiddling didn't help. We're hitting this issue here:

https://github.com/pypa/setuptools/issues/951

That issue is unrelated to this PR, though, so once this PR is set to go, I'll merge it.

@RitwikGupta If you could fix that comma issue, that'd be super. Thank you for working on this!

File: html5lib/constants.py (hunk 1 above)
Comment (gsnedders) [reply html5lib/constants.py]: We switched them all over to lists ages ago to avoid the missing-comma issue.

File: html5lib/constants.py (hunk 2 above)
Comment (gsnedders) [reply html5lib/constants.py]: This is a regression, and why we moved to lists. Presumably just copy/paste from the old branch?

Comment (RitwikGupta) [reply html5lib/constants.py]: This was my fault, I chose the wrong revision when fixing the merge conflicts. Will fix!

**Comment by RitwikGupta** [comment]
I switched over to the lists. I omitted the comma here as stated by @gsnedders, so this should be good to go.

**Comment by willkg** [comment]
Everything looks good, except there's a new merge conflict.

@RitwikGupta Can you fix the merge conflict? If not, I can get to it. I'm really sorry about this.

**Comment by RitwikGupta** [comment]
No problem! Should be good to go here.

**Comment by craigbarnes** [comment]
It seems this attribute has been removed from the spec. It was never supported in Firefox or IE and has been removed from Chrome.

**Pull request merged**